app/helpers/application_helper.rb

```diff
@@ -107,7 +107,7 @@ def process_physical_holding(holding, bib_id, holding_id, is_journal)
       info << content_tag(:h3, location.html_safe, class: 'library-location')
     end
     unless holding['call_number'].blank?
-      cn_browse_link = link_to('[Browse]', "/browse/call_numbers?q=#{holding['call_number_browse']}", class: 'browse-cn', 'data-toggle' => 'tooltip', 'data-original-title' => "Browse: #{holding['call_number_browse']}", title: "Browse: #{holding['call_number_browse']}")
+      cn_browse_link = link_to('<span class="link-text">Browse nearby items</span> <span class="icon-bookslibrary"></span>'.html_safe, "/browse/call_numbers?q=#{holding['call_number_browse']}", class: 'browse-cn', 'data-toggle' => 'tooltip', 'data-original-title' => "Browse: #{holding['call_number_browse']}", title: "Browse: #{holding['call_number_browse']}")
```

Comment:
I have a feeling that the label for the call number browse is something that's going to be changed a couple of times. Could we refer to it via the I18n locale config?